demo/src/components/Cart/Cart.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {Container, Col, Row, Form} from 'react-bootstrap';
+import './Cart.css';
+import {setQuantity} from '../../store/StoreHelpers';
+import Image from 'react-bootstrap/Image';
+import PropTypes from 'prop-types';
+
+/**
+ * Creates a component describing a shopping Cart.
+ * @param {!Object} state The site state
+ * @param {function} setQuantity function to modify quantity of items
+ *      in the global state
+ * @return {!JSX} The component.
+ */
+const CartBase = function Cart({items, setQuantity}) {
+  const itemsRender = [];
+
+  /*
+   * Create the content of the cart display, with one row per item.
+   */
+  for (const [itemID, item] of Object.entries(items)) {
+    if (item.inCart) {
+      itemsRender.push(<Row key={itemID}>
+        <Col><Image className='image-holder' src={item.image}/></Col>
+        <Col><h3>{item.name}</h3><p>{item.description}</p>
+          <Form>
+            <Form.Group>
+              <Form.Label>Quantity</Form.Label>
+              <Form.Control type="number" value={item.quantity}
+                onChange={(event) => {
+                  setQuantity(itemID, +event.target.value);
+                }}/>
+            </Form.Group>
+          </Form>
+        </Col>
+        <Col>{item.cost.toFixed(2)}$</Col>
```

Comment:
I'm not very familiar with react, what this dollar sign means in the end?

Reply:
It's just plaintext, so the display will be something like 92.00$ instead of 92.00